manifests/09_deployment.yaml

```diff
@@ -58,6 +58,8 @@ spec:
           value: "0.0.1-snapshot"
         - name: IMAGE
```

Comment:
Worth renaming IMAGE into `OPENSHIFT_CONTROLLER_MANAGER_IMAGE`

Reply:
I left the name for the following reasons
- it is the main part/image of all the logic managed by the operator
- people might be using this variable in their own scripts, at least I do in mine :) 
- we mention this in our dev docs https://github.com/openshift/enhancements/blob/master/dev-guide/operators.md#how-can-i-test-changes-to-an-openshift-operatoroperandrelease-component